_episodes/l1-04-managing_contributions.md

```diff
@@ -0,0 +1,154 @@
+> - `Forking` a repository means creating a copy of it in your own GitHub account.
+> - This copy is fully under your control, and you can create branches, push new commits, etc., as you would do with any other of your repos.
+> - `fork` is a GitHub concept and not Git.
+> - Forks are related to the original repository, and the number of forks a given repository has can be seen in the upper right corner of the repo page.
+> - If you have some changes in your fork that you want to contribute to the original repo, you open a `pull request`.
+> - You can bring changes from an upstream repository to your local fork.
+>
+>Essentially, the way you use pull requests will depend on what permissions you have for the repository you are contributing to. If the repository owner has not granted you write permission, then you will not be able to create and push a branch to that repository. Conversely, anyone can fork an existing repository and push changes to their personal repository.
+{: .callout}
+
+Now let's take a closer look at those two types of development models;
+
+### 1. Pull request from a branch within a repository
+This type of pull request is used when working with a **shared repository model**. Typically, with this development model, you and your collaborators will have access (and write permission) to a single shared repository. We saw in a previous episode how branches can be used to separate out work on different features of your project. With pull requests, we can request that work done on a feature branch be merged into the `main` branch after a successful review. In fact, we can specify that the work done on our feature branch be merged into *any* branch, not just `main`.
+
+Pull requests can be created by visiting the `Pull request` tab in the repository.
+
+>![Open a pull request]({{ site.baseurl }}/fig/create_pull_request.png "Open a pull request"){:class="img-responsive"}
+{: .callout}
+
+>#### Changing *head* and *base* branch
+>By default, pull requests are based on the parent repository's default branch. You can change both the parent repository and the branch in the drop-down lists. It's important to select the correct order here; the *head branch* contains the changes you would like to make, the *base branch* is where you want the changes to be applied.
+{: .callout}
+
+> ## Now you try
+> Let's revisit our recipe repository. If you have lost your copy of the recipe repository you can download a completed copy [here](https://imperialcollegelondon.github.io/introductory_grad_school_git_course/code/recipe_with_history.zip).
+> 1. Create a remote repository with your recipe repository.
+> 2. Create a new branch with some changes and push the branch to the remote repository.
+> 3. Create a pull request with a suitable title and description to merge the branch containing your changes into the main branch.
+>
+>> ## Solution
```

Comment:
Fair point. Since creating remote repositories and pushing branches isn't the learning objective of this episode, I have linked to the relevant episode in the introductory course.